### DISCUSSION TITLE
Installation Raspberrymatic Fehler "eq3_char_loop" not found

### DISCUSSION THREAD

**Discussion by mickey00001** [OpenCCU/OpenCCU#1688] [🇩🇪 Fragen&Antworten] [answered]
Hallo,
habe gemäß der [Anleitung ](https://github.com/jens-maus/RaspberryMatic/wiki/Installation-Docker-OCI) den RaspberryMatic auf meinem Raspi 3 mit Docker installiert. 

Beim installieren der Voraussetzungen kommt beim Starten von:
**sudo modprobe eq3_char_loop**
die Meldung:
"modprobe: FATAL: Module eq3_char_loop not found in directory /lib/modules/5.10.63-v7+"

Den Ordner an sich gibt es mit folgendem Inhalt:
![image](https://user-images.githubusercontent.com/98711566/151715559-f3e9d55c-e2c8-4a59-b528-07e4e2b8e3bb.png)

**Was kann die Ursache sein?**

**Hardware:**
Raspi 3B
Homematic IP RF-USB-Stick (noch nicht angeschlossen) wurde erst bestellt

**Software:**
RaspberryMatic als DockerContainer installiert
"Home Assistant Supervised" als DockerContainer installiert

Danke Euch für jeden Tipp

**Comment by jens-maus** [comment, marked as answer]
Wurde auch Schritt 4 der Anleitung befolgt? D.h. installation des `pivccu-modules-dkms` paketes? Wenn ja, welche Ausgaben gab es bei der Installation dieses Paketes?

**Comment by mickey00001** [comment]
Im Schritt 4 unter
`sudo apt update`
gab es bei mir eine Auflistung von 138 Paketen die aktualisiert werden mussten. Dies konnte aufgrund einer fehlerhaften Abhängigkeit jedoch noch nicht durchgeführt werden. 

` vlc-bin : Depends: libvlc-bin (= 3.0.16-1+rpi1+rpt1) but 3.0.16-1+rpi1+rpt2 is to be installed
 vlc-plugin-skins2 : Depends: vlc-plugin-qt (= 3.0.16-1+rpi1+rpt2) but 3.0.16-1+rpi1+rpt1 is to be installed`

Die Pakete habe ich dann entfernt.  Danach liefen alle Aktualisierungen durch und im Anschluss lief der Aufruf dann auch fehlerfrei durch.

Danke jens-maus

Grüße mickey00001
